### PR TITLE
chore: Implement release please support.

### PR DESCRIPTION
**Pull request by kinyoklion**

### CODE REVIEW

File: example/eventsource-polyfill.js
Comment (kinyoklion) [inline example/eventsource-polyfill.js:1]: This file is a build artifact and it was updated by building the polyfill.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
-# EventSource [![npm version](http://img.shields.io/npm/v/launchdarkly-eventsource.svg?style=flat-square)](http://browsenpm.org/package/launchdarkly-eventsource)[![Circle CI](https://circleci.com/gh/launchdarkly/js-eventsource/tree/master.svg?style=svg)](https://circleci.com/gh/launchdarkly/js-eventsource/tree/master)[![NPM Downloads](https://img.shields.io/npm/dm/laumchdarkly-eventsource.svg?style=flat-square)](http://npm-stat.com/charts.html?package=launchdarkly-eventsource&from=2015-09-01)[![Dependencies](https://img.shields.io/david/launchdarkly/js-eventsource.svg?style=flat-square)](https://david-dm.org/launchdarkly/js-eventsource)
+# EventSource
+[![ci](https://github.com/launchdarkly/js-eventsource/actions/workflows/ci.yml/badge.svg)](https://github.com/launchdarkly/js-eventsource/actions/workflows/ci.yml)
```
Comment (kinyoklion) [inline README.md:2]: Cleaned up these badges and use the correct CI.

File: CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Repository Maintainers
+* @launchdarkly/team-sdk-js
```
Comment (kinyoklion) [inline CODEOWNERS:2]: This repo should meet the requirements for this code owners. 

I wonder if something isn't quite right in terraform?

Comment (kinyoklion) [reply CODEOWNERS:2]: Yeah, looks like there is a piece missing.

Comment (kinyoklion) [reply CODEOWNERS:2]: This repo has all the correct tags, but it doesn't appear in the query. So I will move forward with this while we investigate in parallel.

**Review by yusinto — APPROVED** [review]

**Pull request merged**